.rubocop.yml

```diff
@@ -81,3 +81,6 @@ Metrics/ParameterLists:
 
 Metrics/PerceivedComplexity:
   Enabled: false
+
+Naming/MemoizedInstanceVariableName: 
```

Comment:
Do we still need this?

Reply:
Yes, I get 
```
lib/bolt/transport/api.rb:19:9: C: Naming/MemoizedInstanceVariableName: Memoized variable @connection does not match method name get_connection. Use @get_connection instead.
        @connection ||= Bolt::Transport::Api::Connection.new(conn_opts, logger, @plan_job)
```
if it's not. I can turn it off locally if it's a cop we want to catch other places. `get_connection` is called from `run_task_job` in the orchestrator transport, and it doesn't seem worth overloading the whole `run_task_job` in the api transport just to give `get_connection` a different name. I also tried renaming `@connection` to `@conn`, but the same error was raised.

Reply:
In general I don't think it's a super important cop. But this method has some smell to it which I think this cop was reasonably picking up on. The method accepts arguments but returns the memoized value which means it probably shouldn't be accepting arguments. IMO we need a larger refactor of this transport/connection probably ditching the "transport" all together and folding this logic directly into the executor/connection.  I think that refactor should probably remain out of scope for this ticket at this point.

If it's easy to fix I would change the function so that it's just a memoized reader for `connection`.